{"time":"2025-03-24T14:53:17.46751142+08:00","level":"INFO","msg":"using version","core version":"0.18.3"}
{"time":"2025-03-24T14:53:17.467548135+08:00","level":"INFO","msg":"created symlink","path":"/home/ckx/minimind/wandb/run-20250324_145317-maduknrc/logs/debug-core.log"}
{"time":"2025-03-24T14:53:17.470214121+08:00","level":"ERROR","msg":"dialing: google: could not find default credentials. See https://cloud.google.com/docs/authentication/external/set-up-adc for more information"}
{"time":"2025-03-24T14:53:17.482556806+08:00","level":"INFO","msg":"created new stream","id":"maduknrc"}
{"time":"2025-03-24T14:53:17.482610729+08:00","level":"INFO","msg":"stream: started","id":"maduknrc"}
{"time":"2025-03-24T14:53:17.48267164+08:00","level":"INFO","msg":"sender: started","stream_id":{"value":"maduknrc"}}
{"time":"2025-03-24T14:53:17.482643726+08:00","level":"INFO","msg":"writer: Do: started","stream_id":{"value":"maduknrc"}}
{"time":"2025-03-24T14:53:17.482651943+08:00","level":"INFO","msg":"handler: started","stream_id":{"value":"maduknrc"}}
{"time":"2025-03-24T14:53:47.484401673+08:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"}
{"time":"2025-03-24T14:54:19.817331174+08:00","level":"INFO","msg":"api: retrying error","error":"Post \"https://api.wandb.ai/graphql\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"}
{"time":"2025-03-24T14:54:49.507985629+08:00","level":"INFO","msg":"stream: closing","id":"maduknrc"}
{"time":"2025-03-24T14:54:49.508113394+08:00","level":"INFO","msg":"handler: closed","stream_id":{"value":"maduknrc"}}
{"time":"2025-03-24T14:54:49.508130644+08:00","level":"INFO","msg":"writer: Close: closed","stream_id":{"value":"maduknrc"}}
{"time":"2025-03-24T14:54:49.508125085+08:00","level":"ERROR","msg":"sender: upsertRun:","error":"failed to upsert bucket: api: failed sending: POST https://api.wandb.ai/graphql giving up after 3 attempt(s): context canceled"}
{"time":"2025-03-24T14:54:49.508170755+08:00","level":"ERROR","msg":"runwork: ignoring record after close","work":{"Record":{"RecordType":{"Request":{"RequestType":{"Defer":{}}}},"control":{"always_send":true}}}}
{"time":"2025-03-24T14:54:49.508679369+08:00","level":"INFO","msg":"sender: closed","stream_id":{"value":"maduknrc"}}
{"time":"2025-03-24T14:54:49.508693058+08:00","level":"INFO","msg":"stream: closed","id":"maduknrc"}
